Sources/SourceKitLSP/SourceKitLSPServer.swift

```diff
@@ -1161,6 +1164,9 @@ extension SourceKitLSPServer {
     if let diagnosticOptions = server.diagnosticProvider {
       await registry.registerDiagnosticIfNeeded(options: diagnosticOptions, for: languages, server: self)
     }
+    if let codeLensOptions = server.codeLensProvider {
+      await registry.registerCodeLensIfNeeded(options: codeLensOptions, for: languages, server: self)
+    }
```

Comment:
I missed this before but I don’t think this does anything if you don’t add `codeLensOptions` to the response returned by `SwiftLanguageService.initialize`.

Reply:
Upon reflection code lenses don't actually need to be dynamically registered, so this call and its related code have been removed.